screens/Registration/AuthenticationScreen/AuthenticationScreen.tsx

```diff
@@ -116,24 +135,32 @@ const AuthenticationScreen: React.FC = () => {
 
         <Divider />
 
-        <InputField
-          label="E-mail*"
-          placeholder="E-mail"
-          onChangeText={setEmail}
-          value={email}
-          ref={secRef}
-          onSubmitEditing={() => thirdRef.current?.focus()}
-        ></InputField>
-
-        <InputField
-          label="Confirm e-mail*"
-          placeholder="Confirm your e-mail"
-          onChangeText={setConfirmEmail}
-          value={confirmEmail}
-          ref={thirdRef}
-        ></InputField>
+        <View>
+          <InputField
+            label="E-mail*"
+            placeholder="E-mail"
+            onChangeText={setEmail}
+            value={email}
+            ref={secRef}
+            onSubmitEditing={() => thirdRef.current?.focus()}
+          ></InputField>
+
+          <InputField
+            label="Confirm e-mail*"
+            placeholder="Confirm your e-mail"
+            onChangeText={setConfirmEmail}
+            value={confirmEmail}
+            ref={thirdRef}
+          ></InputField>
+        </View>
 
-        <LowBar nextScreen="HomeTabs" buttonText="Confirm" authenticate={submitForm} />
+        <View style={[styles.footer, { bottom: insets.bottom }]}>
+          {!keyboardVisible && <LowBar
+            nextScreen="HomeTabs"
+            buttonText="Confirm"
+            authenticate={submitForm}
+          />}
```

Comment:
Why do you want to hide the LowBar when keyboard is on?

Reply:
Sorry for a duplicate solution, but take a look at my code in #165 for how i anchored the LowBar to the bottom. Ideally, you should not wrap the LowBar in a View each time, but rather change the styling for the LowBar itself. Take a look at my solution to see more.

Reply:
The reason I want to hide it is because on IOS, the LowBar was totally fixed but on android the lowbar was fixed until the keyboard was opened and pushed it on top of it